stratum/values.yaml

```diff
@@ -48,3 +48,9 @@ config:
 
 podsecuritypolicy:
   enabled: false
+
+
+# dictionary, key is node name and value is chassis config
+# chassis_config: >
+#    "leaf1" : {}
```

Comment:
Chassis config is a Protobuf text format, which is a multi-line string. But here it seems asking for a map or json `{}`. Can you give an example ?